portfolio/src/main/webapp/script.js

```diff
@@ -139,6 +141,36 @@ function formatDate(epochDate) {
 
 }
 
+/**
+ * Deletes all guestbook comments from the website if the 
+ * correct password is entered.
+ */
+function deleteData() {
+  const enteredPassword = document.getElementById('pwd').value;
+  const requestURL = '/delete-data?password=' + enteredPassword;
+  const request = new Request(requestURL, {method: 'POST'});
+  
+  fetch(request).then((response) => {
+    const passwordMessage = document.getElementById("password-fail");
+    if (response.ok) {
+      passwordMessage.innerText = '';
+      getGuestBook();
```

Comment:
if you add a return after getGuestBook() then you don't need to put the following code in an else block. so it will look like

if (response.ok) {
...
return;
}
response.json().then((jsonResponse) => {...});

which saves you some indentation.